server/src/main/java/io/spine/server/CommandService.java

```diff
@@ -70,14 +75,41 @@ public static Builder newBuilder() {
         return new Builder();
     }
 
-    /** Builds the service with a single Bounded Context. **/
+    /**
+     * Builds the service with a single Bounded Context.
+     */
     public static CommandService withSingle(BoundedContext context) {
         CommandService result = newBuilder()
                 .add(context)
                 .build();
         return result;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>In the original proto definition of this service, this method is unary.
+     * Meaning, for every posted command only a single {@link Ack} is received in response:
+     *
+     * <pre>Ack post(Command);</pre>
+     *
+     * <p>But for every service, gRPC generates several clients: blocking, asynchronous,
```

Comment:
There are two issues with this text.

1. It is pretty much the same for every method which is generated after gRPC. So it would be nice to write it once and "run everywhere" (meaning, reference, not copy).

2. The wording here is a bit harsh. We aren't the authors of gRPC. And for sure we have chosen to use them voluntarily. Therefore, they don't owe us anything. But the impression of this text is that they do.